flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/Expressions.java

```diff
@@ -603,6 +620,15 @@ private static ApiExpression apiCallAtLeastTwoArgument(
         return new ApiExpression(unresolvedCall(functionDefinition, arguments));
     }
 
+    private static ApiExpression apiCallExactlyTwoArgument(
```

Comment:
why is this change necessary?

Reply:
now I noticed that `apiCall` could be used here...
thanks